src/GitReportCL/RunGitReport.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace GitReportCL
```

Comment:
* Do not create public fields `run`, use properties instead
* class name `RunGitReport` should be a noun
* method name `Report`  should be more descriptive